README.md

```diff
@@ -87,8 +85,7 @@ config-version: 2
 
 vars:
     iterable_source:
-        campaign_suppression_list_history: "{{ source('iterable', 'campaign_supression_list_history') }}" 
-        using_old_spelling: true # default false
+        campaign_suppression_list_history_table: "campaign_supression_list_history"
```

Comment:
Same naming thoughts posed in the source package.

`iterable__campaign_suppression_list_history_table`